### PR TITLE
build: added auditors page

### PR DESCRIPTION
**Pull request by aviveiros11**
Satisfies #84 content per the BuildingSync website document.  
Screen Shot 2021-05-06 at 1 18 27 PM

### CODE REVIEW

**Comment by aviveiros11** [comment]
Hey, @kflemin did you get a chance to look at this?

File: bsyncviewer/templates/auditors.html [hunk 1]
```diff
@@ -1,13 +1,85 @@
...
+
+        <div class="row">
+            <div class="col-lg-6">
+                <img src="{% static 'img/lightbulbs.jpg' %}">
+            </div>
+            <div class="col-lg-offset-6">
+                <h4>Track proposed, implemented, and discarded energy conservation measures.</h4>
+                <p style="padding-bottom: 3em">Well-documented technical categories with enumerated measure names for
+                    any use case.</p>
+                <a class="button btn-bg-light col-lg-offset-3" href="/schema">Explore measures</a>
```
Comment (kflemin) [inline bsyncviewer/templates/auditors.html:34]: this is a dead link...where is it supposed to go?

File: bsyncviewer/templates/auditors.html [hunk 2]
```diff
@@ -1,13 +1,85 @@
...
+            Learn how to use BuildingSync with Audit Template Tool</a>
+
+        <h3 style="padding-top: 2em">Documents</h3>
+        <hr>
+        <section id="about" class="hero">
+            <div class="row" style="padding-bottom: 5em">
+                <a href="/practitioners"><div class="hero-item col-lg-6 text-center">
+                    <span class="icon-documents"></span>
+                    <div class="hero-item-content">
+                        <h5><a href="https://buildingsync.net/documents/BuildingSync-on-boarding.pdf">Onboarding Guide</a></h5>
```
Comment (kflemin) [inline bsyncviewer/templates/auditors.html:71]: can you move the document over to this site instead of linking to the old site?

**Review by kflemin — APPROVED** [review]
@aviveiros11 looks good!  just a couple of things to fix...

**Pull request merged**